### DISCUSSION TITLE
关于valaxy的配置问题

### DISCUSSION THREAD

**Discussion by AfterDuang** [YunYouJun/valaxy#197] [Q&A]
我看到站点源文件中有_config.yml和_config.yun.yml,它们和valaxy.config.ts和site.config.ts两个有什么区别吗，另外还要就是求问Waline这种评论系统该怎么弄呢，以及为什么内置的搜索引擎搜不到东西 fuse-list.json这个文件里是空的，把它放进public中似乎也在本地无法搜到~~~

**Comment by YunYouJun** [comment]
站点源文件指的是 `yunyoujun.github.io` 吧，这个是 Hexo 遗留产物，将会被删除。

Waline 见 `valaxy-addon-waline` 插件使用。

`fuse-list.json` 需要使用 `valaxy fuse` 预先生成。

**Comment by AfterDuang** [comment]
<img src="/images/GameNote/0/WorldCeatorDoc1.png" width="100%" height="100%" style="zoom:150%">
我在文章中写这种格式插入图片的是不是现在会报错了

**Comment by AfterDuang** [comment]
html格式插入图片

**Reply by YunYouJun** [reply]
html 不会

**Reply by AfterDuang** [reply]
怎么样构建fuse的本地缓存呢，我直接npm run fuse 是没有效果的，捣鼓了蛮久了QAQ

**Reply by YunYouJun** [reply]
添加脚本命令 scripts package.json

https://github.com/YunYouJun/valaxy/blob/bdb2d51289f71ec856d45723e6ab80d2a6e1b868/demo/yun/package.json#L11

再 `npm run fuse`。

**Reply by YunYouJun** [reply]
https://valaxy.site/guide/third-party#%25E6%259C%25AC%25E5%259C%25B0%25E6%2590%259C%25E7%25B4%25A2%25EF%25BC%2588%25E5%259F%25BA%25E4%25BA%258E-fuse-js%25EF%25BC%2589

**Reply by AfterDuang** [reply]
文档上的内容我都做了呀，但是生成的fuse-list.json这个文件中只有一个 [] 这样的方括号

**Reply by YunYouJun** [reply]
请提供可复现最小仓库。

**Reply by AfterDuang** [reply]
https://github.com/AfterDuang/ValaxySourceCode/tree/master
